content/articles/understanding-blockchain-as-a-service/index.md

```diff
@@ -0,0 +1,87 @@
+### Blockchain as a Service
+
+### Introduction
+The blockchain is the new buzzword in town and everyone seems to be talking about it. But how do businesses actually use the technology without having to build their own blockchains? Blockchain as a service (BaaS) enables companies to develop applications on various blockchains— such as Ethereum, Ripple or NEO —without having to code from scratch. Clients benefit from BaaS because it allows for faster application development, lower maintenance costs, and faster adoption of Blockchain technology.
```

Comment:
`Clients benefit from BaaS because it allows for faster application development, lower maintenance costs, and faster adoption of Blockchain technology.`

Unless you compare this with the existing SaaS models or other models. You can't tell this.

Please justify this in the upcoming topics below.